src/foundation.cs

```diff
@@ -3173,9 +3172,12 @@ public interface NSExpression : NSSecureCoding, NSCopying {
 		[Static, Export ("expressionForFunction:arguments:")]
 		NSExpression FromFunction (string name, NSExpression[] parameters);
 
+		[Static, Export ("expressionWithFormat:")]
+		NSExpression FromFormat (string expressionFormat);
+
 		[Static, Export ("expressionWithFormat:argumentArray:")]
-		NSExpression FromFormat (string format, NSExpression [] parameters);
-		
+		NSExpression FromFormat (string format, NSObject [] parameters);
```

Comment:
I believe those arguments there are for the subclass that takes a function and not a format. Lets TDD to get this right. I'll write all the tests for the expected behaviour and change the bindings accordingly.